packages/api/src/command/medical/patient/create-patient.ts

```diff
@@ -45,7 +51,6 @@ export const createPatient = async (
   // validate facility exists and cx has access to it
   await getFacilityOrFail({ cxId, id: facilityId });
 
-  const requestId = uuidv7();
```

Comment:
nit: nbd but why are we moving this to the route?

Reply:
If its because of the ticket 1829 we could just leave this as is and then move to http or just do it now

Reply:
Could just initialize in `runInitialPatientDiscoveryAcrossHies`. Only place being used

Reply:
we're moving it to the route as this is where the requestId gets established, and all methods from the route should inherit this ID. Essentially it's as far up the chain as we can go.

If we're setting it lower down the chain (i.e. command or the hie command), it loses some of its value.

Reply:
I moved it to the new HIE functions.

Reply:
@leite08 @Orta21  that work for you?